src/mmw/sass/components/_modals.scss

```diff
@@ -31,7 +31,7 @@
 .modal-basic {
 
   .modal-content {
-    padding: 1rem;
+    padding: 3rem;
```

Comment:
Why did the primary action button change from "Make Public" to "Confirm" in the second screenshot?